frontend/server/src/Controllers/Clarification.php

```diff
@@ -286,14 +286,16 @@ public static function apiUpdate(\OmegaUp\Request $r): array {
 
         // Validate request
         $r->ensureInt('clarification_id');
```

Comment:
```suggestion
        $clarificationId = $r->ensureInt('clarification_id');
```

así se puede eliminmar el uso de `$r['clarification_id']` por consistencia. o incluso se puede dar inline directamente en L303 para evitar una declaración.